client/src/components/scrum/index.jsx

```diff
@@ -4,6 +4,7 @@ import { Link } from "react-router-dom";
 export default class Scrum extends Component {
   render() {
     const id = this.props.match.params.scrumId;
+    console.log(id)
```

Comment:
:crying_cat_face: 
seriously?
You commit `console.log`  :disappointed:

Reply:
remove change